### ISSUE TITLE
implement dom0 -> right click -> VM settings / easy way to configure updatevm setting

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#1165]
Currently `dom0 -> right click -> VM settings` is grayed out. Would it make sense to implement that?

To my knowledge, there is currently no way to set dom0 updatevm, i.e. `qubes-prefs --set updatevm whonix-gw` by using the gui.

This is related to...
`high-level target: templates should default update over Tor`:
#1159

If #1159 is not going to be implemented, this ticket could be closed or set to "patches welcome".

**Comment by marmarek** [comment]
It is possible: menu "system" -> global settings. But indeed it isn't
intuitive. And even if one find that settings, it isn't clear what it
does...

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?

**Comment by marmarek** [comment]
Maybe "settings" for dom0 should simply open global settings?

**Comment by adrelanos** [comment]
Sounds good to me.

**Comment by adrelanos** [comment]
Has this been reverted? Is there a regression here?

The right click menu in QVMM on dom0 shows most grayed out for me. Only update, qubes windows tools installation and logs is clickable.

Please re-open.

//cc @marmarta

**Comment by marmarta** [comment]
yes, it has been long reverted, I think. We could add global config there, but I'm not sure if it's that easy to find... Global Settings are linked in System menu and toolbar in manager, and I think it's a generally more reasonable place.

**Comment by marmarta** [comment]
And I think it's better to think about stuff like "how updates are made" from perspective of global settings rather than from perspective of "dom0 settings", especially as with introduction of gui domain it gets more and more complex to look at those settings from dom0 perspective.

**Comment by ben-grande** [comment]
@marmarta I didn't understand what is missing here. It was implemented in Global config right? I don't think it should be in dom0 vm settings, it would be unusable from a restricted GUI domain.

**Comment by marmarta** [comment]
Yeah, I think that the explanation here is good and the decision is correct.
